test/integration/serverless/run.sh

```diff
@@ -104,7 +112,22 @@ function remove_stack() {
 trap remove_stack EXIT
 
 # deploy the stack
-serverless deploy --stage "${stage}"
+if [ "$ARCHITECTURE" == "amd64" ]; then
+    export DEPLOYMENT_REGION=sa-east-1
+    export DEPLOYMENT_BUCKET=integration-tests-deployment-bucket
+    echo "Deploying stack with $ARCHITECTURE architecture to $DEPLOYMENT_REGION, using the $DEPLOYMENT_BUCKET"
+
+    # we use the architecture name amd64, while AWS requires it to be listed as x86_64
+    export DEPLOYMENT_ARCHITECTURE=x86_64
+    serverless deploy --stage "${stage}"
+else 
+    export DEPLOYMENT_REGION=eu-west-1
```

Comment:
Why do we use a different region for ARM?

Reply:
sa-east-1 doesn't support arm for lambda functions

Reply:
Would it be simpler to just test both in eu-west-1?

Reply:
That would be easier, we could use the same deployment bucket that way. I dont see any downsides to this?